spec/models/concerns/hyrax/collection_nesting_spec.rb

```diff
@@ -15,6 +17,13 @@ def update_index
           true
         end
 
+        def before_save
+          false
```

Comment:
Should this be true like the return values for the other callbacks in this class?

Reply:
no, making this true would cause the other specs to not run. But I am adding a new test to verify that this being true causes the nesting reindexer to not get called.